rpisec/rpis_camera.py

```diff
@@ -11,7 +11,12 @@
 from queue import Queue
 from .exit_clean import exit_error
 from datetime import datetime
+import json
```

Comment:
Is this used?

Reply:
No, I used it during developments, but its no more used. I will remove this